test/Controller/JobAdvertsControllerTest.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace test\eLife\Journal\Controller;
+
+use GuzzleHttp\Psr7\Request;
+use GuzzleHttp\Psr7\Response;
+use Traversable;
+
+final class JobAdvertsControllerTest extends PageTestCase
+{
+    /**
+     * @test
+     */
+    public function it_displays_the_job_adverts_page()
+    {
+        $client = static::createClient();
+
+        $crawler = $client->request('GET', $this->getUrl());
+
+        $this->assertSame(200, $client->getResponse()->getStatusCode());
+        $this->assertSame('eLife Jobs', $crawler->filter('.content-header__title')->text());
+        $this->assertSame('Still awaiting final text, this test should fail', $crawler->filter('.content-header__impact-statement')->text());
```

Comment:
Expecting this test to fail until wording is finalised.

(We can always remove this and https://github.com/elifesciences/journal/pull/757/files#diff-9b91ec359d378adcb8864a3b75f22950R67 if we want to merge before that happens.)